// Copyright (c) 2022 Boston Dynamics, Inc.  All rights reserved.
//
// Downloading, reproducing, distributing or otherwise using the SDK Software
// is subject to the terms and conditions of the Boston Dynamics Software
// Development Kit License (20191101-BDSDK-SL).

syntax = "proto3";

package bosdyn.api;

option java_outer_classname = "IREnableDisableServiceProto";

import "bosdyn/api/ir_enable_disable.proto";

service IREnableDisableService {
    rpc IREnableDisable(IREnableDisableRequest) returns (IREnableDisableResponse) {}
}
